pkg/devspace/helm/v3/client.go

```diff
@@ -126,7 +127,12 @@ func (c *client) InstallChart(releaseName string, releaseNamespace string, value
 	}
 
 	args = append(args, helmConfig.UpgradeArgs...)
-	_, err = c.genericHelm.Exec(args, helmConfig)
+	output, err := c.genericHelm.Exec(args, helmConfig)
+
+	if helmConfig.Output {
+		c.log.Infof("Helm Output '%s'", output)
```

Comment:
Since there is an output of `[info]   Execute 'helm upgrade ...` just before, I don't think we need a descriptive message here anymore as it should be clear the output corresponds to the helm command. So you could just do:

```
if helmConig.DisplayOutput {
  c.log.Write(output)
}
```